docs/simplecard.md

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
You need to add `simplecard` to `sidebars.json` in order for it to render on the sidebar.